server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+      operationId: getApplications
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Applications'
+  /functionalities/{application}/tasks:
+    parameters:
+      - name: application
+        in: path
+        description: The name of an application
+        required: true
+        schema:
+          $ref: '#/components/schemas/Applications'
+    # GET /ssot/:id
+    get:
+      tags:
+        - RPA-Functionalities
+      summary: Gets all the tasks to execute for an application
+      operationId: getTasks
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Tasks'
+        404:
+          description: Not Found
+  #### Robots ####
+  /robots/{robotId}:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /ssot/:id
+    get:
+      tags:
+        - Robots
+      summary: Get a robot with a specific id
+      operationId: getSpecificRobot
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Robots'
+    # // POST /overwriteRobot/78d09f66d2ed466cf20b06f7
+    put:
+      tags:
+        - Robots
+      summary: Overwrite the existing robot with an updated one
+      operationId: overwriteRobot
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Robots'
+        description: updated Robot object
+        required: true
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+  /robots/{robotId}/robotCode:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /parser/getForId/:robotId
+    get:
+      tags:
+        - Robots
+      summary: Get the robot framework code of a specific robot
+      operationId: getRobotCode
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RobotCode'
+  /robots/{robotId}/robotName:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /renameRobot?id=78d09f66d2ed466cf20b06f7&newName=Bot+Browser
```

Comment:
shouldnt this comment be different when its a patch request?

Reply:
The purpose of the comments is not to describe the endpoints. All these comments are refering to the "old" call respectively. This is only in here to help with #265 and should be deleted once the new api schema is integrated in our product.